cirq/experiments/fidelity_estimation_test.py

```diff
@@ -240,3 +248,94 @@ def test_least_squares_xeb_fidelity_from_probabilities():
     np.testing.assert_allclose(np.sum(np.array(r_log_math) ** 2), 0.0, atol=1e-2)
 
     np.random.set_state(prng_state)
+
+
+def test_sample_2q_xeb_circuits():
```

Comment:
Are circuits of depth 50 necessary to test this functionality? Is it expensive to run those tests? I'm asking because Cirq tests are not becoming any shorter these days...

Reply:
It "only" takes 948 ms. I'll pare it down. `cycle_depths = np.arange(3, 20, 6)` is 306 ms